cmake/configure/configure_20_boost.cmake

```diff
@@ -146,7 +146,10 @@ endmacro()
 
 
 macro(feature_boost_configure_external)
-  enable_if_supported(BOOST_CXX_FLAGS "-Wno-unused-local-typedefs")
+  #
+  # Avoid a number of warnings:
+  #
+  enable_if_supported(DEAL_II_CXX_FLAGS "-Wno-unused-local-typedefs")
```

Comment:
Does this end up in the compile options for `deal::dealii_debug` and `dealii::dealii_release`?

Reply:
No, I have kept the `dealii::dealii_(debug|release)` targets free of any compiler and linker flags - simply because this would imply that a user project cannot override those flags any more...

Hence the continued use of the `deal_ii_setup_cached_variables()` or the `deal_ii_setup_target()` macro...

Ultimately, I want to remove all of these warning toggles from DEAL_II_CXX_FLAGS* and have a separate DEAL_II_WARNING_FLAGS* variable.